### DISCUSSION TITLE
Disable SIMD (Extra Post)

### DISCUSSION THREAD

**Discussion by giscus** [phil-opp/blog_os#1330] [Post Comments (translated)]
# Disable SIMD (Extra Post)

单指令多数据 指令允许在一个操作符（比如加法）内传入多组数据，以此加速程序执行速度。x86_64 架构支持多种SIMD标准：

https://os.phil-opp.com/zh-CN/disable-simd/

**Comment by Rogerskelamen** [comment]
感谢分享这么优质的内容💕
